### DISCUSSION TITLE
Web - production (nginx)

### DISCUSSION THREAD

**Discussion by Silko5** [simolus3/drift#2740] [Q&A] [answered]
Hi,

I have a problem with deploying my flutter web app to production server. Everything works perfectly in local environment, but when I deploy the app to production I think that the database is not generated. In Chrome if I check dev tools -> Application ->  IndexedDB, locally I see my database there but in production it is not there. In production I only get a lot of uncaught errors in main.dart.js and I don't have ideas anymore how to debug this to get it to work. Any ideas where to start?

If it helps. Console log for each uncaught error is:
Uncaught Error
    at Object.oU (main.dart.js:7706:24)
    at V6.a6E (main.dart.js:55201:51)
    at V6.wp (main.dart.js:55204:23)
    at main.dart.js:55098:19
    at aRg.a (main.dart.js:6263:62)
    at aRg.$2 (main.dart.js:47533:14)
    at Object.u (main.dart.js:6249:10)
    at NF.ayx (main.dart.js:55102:10)
    at NF.kd (main.dart.js:55092:19)
    at ar3.$1 (main.dart.js:57447:10)

**Comment by simolus3** [comment]
Could you check that the wasm file gets served with the right content type (`application/wasm`)? Are there any additional errors showing up in the console?

**Comment by Silko5** [comment]
![image](https://github.com/simolus3/drift/assets/6794536/4bf99a04-244e-4f71-b8a0-a010ca4f0cf1)

Here is print screen from dev tools. If I checked at the right place it is not the right content type. Also you can see part of the console with the errors. They are all the same.

EDIT: 
I must add that canvaskit.wasm gets served as application/wasm.

**Comment by Silko5** [comment, marked as answer]
Thank you very much @simolus3. You pointed me in the right direction. I fixed it by adding this to nginx.conf:
```
types {
    application/wasm wasm;
 }
```
